dashboards/workspace/workspace_report_for_admin.sp

```diff
@@ -0,0 +1,171 @@
+dashboard "workspace_report_admin" {
+
+  title         = "Turbot Guardrails Workspace Report for Admin"
+  documentation = file("./dashboards/workspace/docs/workspace_report_for_admin.md")
+
+  tags = merge(local.workspace_common_tags, {
+    type     = "Report"
+    category = "Summary"
+  })
+
+  # Analysis
+  container {
+    card {
+      sql   = query.workspaces_count.sql
+      width = 2
+      href  = dashboard.workspace_report.url_path
+    }
+
+    card {
+      sql   = query.accounts_total.sql
+      width = 2
+      href  = dashboard.workspace_account_report.url_path
+    }
+
+    card {
+      sql   = query.resources_count.sql
+      width = 2
+    }
+  }
+
+  container {
+    title = "Stacks Summary"
+    table {
+      width = 8
```

Comment:
Can we keep the width at either `12` or have two tables side by side with a width set to `6` each? Currently, the tables don't cover the full page, which doesn't look good.